package.json

```diff
@@ -52,6 +53,7 @@
     "html-parse-string": "^0.0.9",
     "i18next": "^23.5.1",
     "i18next-http-backend": "^2.2.2",
+    "luxon": "^3.4.3",
```

Comment:
Ciekawostka, sprawdziłem jak się ma wersja paczki @types vs wersja samej paczki.

I wersja major.minor @types/<package> musi pokrywać się z major.minor typowanej paczki (czyli w tym przypadku, powinno być zainstalowane @types/luxon w wersji conajmniej 3.4.0, a nie jest, więc teoretycznie może być jakaś niezgodność typów).

Source: https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

Reply:
Typowo nowa wersja minor powinna być wstecznie kompatybilna więc mogą być jakieś featury które biblioteka ma a typy nie, ale nie powinno się raczej zdarzyć odwrotnie, więc chyba bym to olał i zostawił obie na najnowsze, tak jak jest.